spring-native-aws-lambda-function/pom.xml

```diff
@@ -169,19 +163,14 @@
             <artifactId>native-maven-plugin</artifactId>
             <configuration>
               <mainClass>com.coffeebeans.springnativeawslambda.Application</mainClass>
-              <buildArgs>
-                --no-fallback
+              <buildArgs combine.children="append">
+                <buildArg>--verbose</buildArg>
+                <buildArg>--no-fallback</buildArg>
+                <buildArg>--enable-preview</buildArg>
+                <buildArg>--strict-image-heap</buildArg>
+                <buildArg>-H:+ReportExceptionStackTraces</buildArg>
               </buildArgs>
             </configuration>
-            <executions>
```

Comment:
@andrew2184 Without this, the `maven-assembly-plugin` fails to include the `spring-native-aws-lambda-function` executable. I think because the compile goal forks the process.